### PR TITLE
Add tag categorization

### PR DESCRIPTION
**Pull request by lilrosalyn**
Adds:
- **!tag setsection <tag name> <section>**
- **!tag set <section> <tag name> <tag content>**
- **!tags** shows tags in categories. Tags without categories will appear in the category **Uncategorized**

### CODE REVIEW

**Comment by jellz** [comment]
Could you propose some categories for the existing tags? I noticed Informative in one command example but all tags are informative lol

**Comment by lilrosalyn** [comment]
> Could you propose some categories for the existing tags? I noticed Informative in one command example but all tags are informative lol

I purely just put "informative" as an example word, there aren't any hard coded categories. Anyways, there could be ones like `panel`, `permissions`, `world`, `discord`, etc.

File: src/command/tag/info.ts [hunk 1]
```diff
@@ -45,9 +45,10 @@ export default class TagInfoCommand extends MinehutCommand {
 		if (tag.aliases.length > 0)
 			embed.addField(
 				'Aliases',
-				tag.aliases.map(a => `\`${a}\``).join(', '),
+				tag.aliases.map((a: any) => `\`${a}\``).join(', '),
```
Comment (draem) [inline src/command/tag/info.ts]: don't use any please
~draem

File: src/command/tag/list.ts [hunk 2]
```diff
@@ -22,9 +22,20 @@ export default class TagListCommand extends MinehutCommand {
 		const embed = new MessageEmbed();
 		embed.setColor('ORANGE');
 		embed.setTitle(`Showing ${tags.length} tags`);
-		embed.setDescription(
-			truncate(tags.map(t => `\`${t.name}\``).join(', '), { length: 2048 })
-		);
+		const fields: any = {};
```
Comment (draem) [inline src/command/tag/list.ts]: don't use any please

File: src/command/tag/list.ts [hunk 3]
```diff
@@ -22,9 +22,20 @@ export default class TagListCommand extends MinehutCommand {
...
+		for (const tag of tags) {
+			if (!fields[tag.section || 'Uncategorized'])
+				fields[tag.section || 'Uncategorized'] = [];
+			fields[tag.section || 'Uncategorized'].push(tag);
+		}
+		for (const name in fields) {
+			const field = fields[name];
+			const mapped = truncate(
+				field.map((t: any) => `\`${t.name}\``).join(', '),
```
Comment (draem) [inline src/command/tag/list.ts]: don't use any please

**Review by draem — CHANGES_REQUESTED** [review]

**Comment by theminecoder** [comment]
Please merge master into this and update the commands to handle the updated roles

**Comment by lilrosalyn** [comment]
That should be everything. If I've missed something, please comment :)

**Review by theminecoder — APPROVED** [review]
Looks good. Just giving sr mods/managers some heads up before this is merged so tags can immediately be put in the right sections.

**Pull request merged**